komodoenv/creator.py

```diff
@@ -69,7 +64,6 @@ def remove_file(self, path):
     def venv(self):
         self.print_action("venv", "using {}".format(self.srcpy.executable))
 
-        ld_library_path = os.environ.get("LD_LIBRARY_PATH")
         env = {"LD_LIBRARY_PATH": str(self.srcpath / "root" / "lib"), **os.environ}
```

Comment:
the assigned variable is not used --- should it be?  In my pr i've just removed it....

Reply:
Should be fine. I think I previously modified the current process' environment rather than passing a modified copy to the subprocess.